vit/application.py

```diff
@@ -740,6 +764,11 @@ def task_action_modify(self):
             self.activate_command_bar('modify', 'Modify: ', {'uuid': uuid})
             self.task_list.focus_by_task_uuid(uuid, self.previous_focus_position)
 
+    def task_action_modify_all(self):
+        currentviewfilter = self.active_view_filters()
```

Comment:
can we do `current_view_filter`? easier to read.

Reply:
Done!